### ISSUE TITLE
Preact Types

### ISSUE THREAD

**Issue by maxpavlov9991** [storybookjs/storybook#13126]
Hello!
I'm working on a project which uses preact & typescript. For about 2 days ago i updated project storybook to version 6:

"@storybook/preact": "^6.0.28",

but when I started using new features added in version 6, i discovered that i have the same problem as here

https://github.com/storybookjs/storybook/issues/11916

The problem in upper link resolved, but for React.

Is there anything similar to:

`import { Story, Meta } from '@storybook/preact/types-6-0'`

or maybe there is some other solution?

Thanks for your answers.

**Comment by shilman** [comment]
Closing as duplicate to #11845

PRs welcome!

**Comment by geisterfurz007** [comment]
Quick heads up: The Preact types for CSF were released in `@storybook/preact@6.2.0-alpha.31` so the following works:

```ts
import { h } from 'preact';
import { Meta, Story } from '@storybook/preact/types-6-0';

const meta: Meta = {
  title: 'Example/Example',
  argTypes: {
    label: {control: "text"},
  },
};

export default meta;

const Template: Story<{ example: string }> = (args) => <button>{ args.example }</button>;
export const Example = Template.bind({});
Example.args = {
  example: 'Example',
};
```

*Make sure to use `npx sb upgrade --prerelease` to update all storybook dependencies at once; I only updated `@storybook/preact` at first and storybook failed to load stories in the browser.*
